packages/browser/package.json

```diff
@@ -87,6 +88,7 @@
     "license-checker": "^25.0.1",
     "lodash.clonedeep": "^4.5.0",
     "node-jose": "^1.1.3",
+    "oidc-client": "^1.10.1",
```

Comment:
This should be replaced by a dependency on oidc-dpop-client-browser, which should export the functions you need. (And thus, there should be no changes to `package-lock.json`.)